pages/pricing/_enterprise.html

```diff
@@ -0,0 +1,9 @@
+<div id="pricing-faqs" class="row row-page-header">
+  <div class="col-xs-12">
+    <h2 style="padding-top: 27px;">Enterprise Agreements</h2>
+    <p>Need to customize our <a href="/terms">Terms of Service</a>? Download our terms as a <a href="#">Microsoft Word
+      File</a>, but please note that we require an Enterprise Service Plan for all custom contracts. The Enterprise
+      Service Plan includes a faster Service Level Agreement, more user licenses, and other benefits.</p>
+    <p>For more information, <a href="/contact"> Contact Us</a>.</p>
+  </div>
+</div>
```

Comment:
Should this be removed from this PR?

Reply:
It's not being called in the page build. I think we should be fine. I can remove it if you'd like.

Reply:
Probably best to remove to keep the PRs clean and self-contained.